### ISSUE TITLE
aria-checked on option: how to expose in AX API?

### ISSUE THREAD

**Issue by spectranaut** [w3c/core-aam#270]
We allow aria-checked on role=option, see https://w3c.github.io/aria/#option and https://w3c.github.io/aria/#aria-checked.

We also allow BOTH aria-checked AND aria-selected on a role=option.

The first problem is that you can't expose aria-checked the way it is currently defined to be exposed (see: https://w3c.github.io/aria/core-aam/#ariaCheckedTrue) -- because in AX API the checked state is exposed though AXValue, but AXValue (on chrome at least) is the same as the accessible name as the role=option.

You could maybe expose aria-checked using AXSelected, but then you need to decide on a conflict resolution between aria-checked and aria-selected, and cannot expose them both, which the spec explicitly allows.

**Comment by spectranaut** [comment]
Someone had made an issue on [Webkit](https://bugs.webkit.org/show_bug.cgi?id=220255) and [Chrome](https://issues.chromium.org/issues/450044214) about this.
